manifests/init.pp

```diff
@@ -37,9 +37,7 @@
 
   $collectd_version_real = pick_default($facts['collectd_version'], $minimum_version)
 
-  class { '::collectd::install':
-    package_install_options => $package_install_options,
-  }
+  class { '::collectd::install': }
```

Comment:
this should be converted to 'contain collectd::install'?

Reply:
Yes it should, but I intend to do this in later PR as part of data type changes to init.pp